monkey/monkey_island/Pipfile

```diff
@@ -34,6 +34,7 @@ pydantic = "*"
 egg-timer = "*"
 pyyaml = "*"
 semver = "==2.13.0"
+shell = "*"
```

Comment:
Curious why we need this? I didn't need it in my testing last week.

Reply:
I'm guessing it has something to do with `pipenv shell`... we probably don't need it